orchagent/main.cpp

```diff
@@ -255,18 +379,21 @@ int main(int argc, char **argv)
     attr.value.ptr = (void *)on_switch_shutdown_request;
     attrs.push_back(attr);
 
+    // Instantiate database connectors
+    DBConnector appl_db("APPL_DB", 0);
+    DBConnector config_db("CONFIG_DB", 0);
+    DBConnector state_db("STATE_DB", 0);
+
+    // Get switch_type
+    getCfgSwitchType(&config_db, gMySwitchType);
```

Comment:
Return value not handled?

Reply:
Yes. But instead of handling error outside, I'll make this function "void" and inside the function for any value other than "voq" or "fabric" gMySwitchType will be set "switch" (to indicate regular switch).

Reply:
Fixed